app/views/admin/depositors_search/search.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :breadcrumbs do %>
+  <%= render Elements::BreadcrumbNavComponent.new do |component| %>
+    <% component.with_breadcrumb(text: 'Dashboard', link: dashboard_path) %>
+    <% component.with_breadcrumb(text: 'Admin dashboard', link: admin_dashboard_path) %>
+    <% component.with_breadcrumb(text: 'Get SUNet IDs of depositors') %>
+  <% end %>
+<% end %>
+
+<div class="d-flex justify-content-end">
+  <%= render Admin::FunctionDropdownComponent.new %>
+</div>
+
+<%= render Elements::HeadingComponent.new(level: :h1, text: 'Get SUNet IDs of depositors', classes: 'h2 mb-4') %>
+
+<%= turbo_frame_tag 'depositors-search-form' do %>
+  <%= form_with model: @depositors_search_form, url: search_admin_depositors_search_path, method: :get, class: 'mb-5', data: { turbo_frame: 'depositors-search' } do |form| %>
```

Comment:
Is this turbo frame necessary?

Reply:
I'd removed it locally and failed to push. So no!